entrypoints/content/utils/hide_works.ts

```
@@ -0,0 +1,29 @@
+      `[data-tag-name="${tagName}"]`
+    );
+
+    await Promise.all(
+      Array.from(works).map((element) => {
+        const liElement = element.closest('li');
+        if (!liElement) return;
+        liElement.style.display = 'none';
+      })
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)